addons/confirm-actions/userscript.js

```diff
@@ -8,6 +8,8 @@ export default async function ({ addon, console, msg }) {
         e.target.closest("[class*='share-button_share-button']:not([class*='is-shared']), .banner-button")
       ) {
         cancelMessage = msg("share");
+      } else if (addon.settings.get("projectunsharing") && e.target.closest(".media-stats a.unshare")) {
```

Comment:
This has a different selector than the share button. Should the share button use `.media-stats a.shared`?

Reply:
There is no share button in My Stuff?

Reply:
Ah yes, I thought this was in the editor whoops